sycl/include/CL/sycl/detail/helpers.hpp

```diff
@@ -17,6 +17,7 @@
 #include <CL/sycl/detail/type_traits.hpp>
 
 #include <memory>
+#include <numeric> // std::bit_cast
```

Comment:
Isn't `std::bit_cast` defined in `<bit>`?

Reply:
It is, but ```<bit>``` library is defined since C++20 and it is part of ```<numeric>``` library. DPC++ runtime builds with C++ std libraries w/o C++20 support.

Reply:
Makes sense, LGTM.